frontend/src/metabase/dashboard/reducers.js

```diff
@@ -163,7 +166,7 @@ const dashcards = handleActions(
       }),
     },
     [SET_MULTIPLE_DASHCARD_ATTRIBUTES]: {
-      next: (state, { payload: dashcards }) => {
+      next: (state, { payload: { dashcards } }) => {
```

Comment:
Changed this is it can be a named attribute of the object which is more semantic and typical in Redux. The payload is now `{ dashcards: [card1, card2, ...] }` instead of just being the array itself.